app/views/catalog/_osd_or_embed_default.html.erb

```diff
@@ -1,5 +1,8 @@
+<% @viewer = Viewer.find_by(exhibit: @exhibit) %>
```

Comment:
@csbailey5t Is the `@viewer` variable set here used by another partial somewhere else?  If you change this from `@viewer` to `viewer` (and change the line below) do things break elsewhere?